networkx/algorithms/euler.py

```diff
@@ -15,10 +21,15 @@
 
 from ..utils import arbitrary_element
 
+from collections import deque
+
 __author__ = """\n""".join(['Nima Mohammadi (nima.irt[AT]gmail.com)',
-                            'Aric Hagberg <hagberg@lanl.gov>'])
+                            'Aric Hagberg <hagberg@lanl.gov>',
```

Comment:
I'm not clear on this spec.  I understand that we're going to be slowly moving through the code shifting ```__author__``` lines to comments. But I never saw a discussion of how that should appear.

Are we going to list authors in a comment line, or list them as part of a copyright statement? If we list them as part of a copyright, how do we handle the year, etc...  @hagberg I think you have talked to lawyer types about this sort of thing. How should we annotate authors in the source?